src/library/Uno.Themes.WinUI.Markup/Theme/PipsPager.cs

```diff
@@ -0,0 +1,247 @@
+								public static ResourceValue<double> Default => new("MaterialPipsPagerNavigationButtonVisualStateEllipseWidth", true);
+							}
+
+							public static class Fill
+							{
+								[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerNavigationButtonVisualStateEllipseFill")]
+								public static ResourceValue<Brush> Default => new("MaterialPipsPagerNavigationButtonVisualStateEllipseFill", true);
+
+								[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerNavigationButtonVisualStateEllipseFillPointerOver")]
+								public static ResourceValue<Brush> PointerOver => new("MaterialPipsPagerNavigationButtonVisualStateEllipseFillPointerOver", true);
+
+								[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerNavigationButtonVisualStateEllipseFillPressed")]
+								public static ResourceValue<Brush> Pressed => new("MaterialPipsPagerNavigationButtonVisualStateEllipseFillPressed", true);
+
+								[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerNavigationButtonVisualStateEllipseFillFocused")]
+								public static ResourceValue<Brush> Focused => new("MaterialPipsPagerNavigationButtonVisualStateEllipseFillFocused", true);
+
+								[ResourceKeyDefinition(typeof(Brush), "MaterialPipsPagerNavigationButtonVisualStateEllipseFillUnfocused")]
+								public static ResourceValue<Brush> Unfocused => new("MaterialPipsPagerNavigationButtonVisualStateEllipseFillUnfocused", true);
+							}
+						}
+
+						public static class PreviousPageData
+						{
+							[ResourceKeyDefinition(typeof(Geometry), "PipsPagerPreviousPageButtonData")]
+							public static ResourceValue<Geometry> Default => new("PipsPagerPreviousPageButtonData", true);
+						}
+
+						public static class NextPageData
+						{
+							[ResourceKeyDefinition(typeof(Geometry), "PipsPagerNextPageButtonData")]
+							public static ResourceValue<Geometry> Default => new("PipsPagerNextPageButtonData", true);
+						}
+
+						public static class Height
+						{
+							[ResourceKeyDefinition(typeof(double), "PipsPagerNavigationButtonHeight")]
+							public static ResourceValue<double> Default => new("PipsPagerNavigationButtonHeight", true);
+						}
+
+						public static class Width
+						{
+							[ResourceKeyDefinition(typeof(double), "PipsPagerNavigationButtonWidth")]
+							public static ResourceValue<double> Default => new("PipsPagerNavigationButtonWidth", true);
+						}
+
+						public static class BorderThickness
+						{
+							[ResourceKeyDefinition(typeof(Thickness), "PipsPagerNavigationButtonBorderThickness")]
+							public static ResourceValue<Thickness> Default => new("PipsPagerNavigationButtonBorderThickness", true);
+						}
+
+						public static class Padding
+						{
+							[ResourceKeyDefinition(typeof(Thickness), "PipsPagerNavigationButtonPadding")]
+							public static ResourceValue<Thickness> Default => new("PipsPagerNavigationButtonPadding", true);
+						}
```

Comment:
No need for a Default nested property and these two should be moved into the NavigationButton class as Height, Width, BorderThickness, and Padding properties